doc-deploy-dev/action.yml

```diff
@@ -34,11 +67,29 @@ runs:
       shell: bash
       run: rm -rf dev/ && mkdir dev/
 
-    - name: "Download the HTML documentation artifact"
+    - name: "Download the HTML documentation artifact from the same workflow"
       uses: actions/download-artifact@v3
+      if: ${{ inputs.workflow == "" }}
+      with:
+        name: ${{ inputs.doc-artifact-name }}
+        path: dev
```

Comment:
This branching can be deleted if we are happy with using ``dawidd6/action-download-artifact@v2`` for everything... I think we can be confident enough we should not mess it up.

Reply:
@jorgepiloto  What about this?

Reply:
I'm just going to use that action for everything.